Backend/credentials/models.py

```diff
@@ -24,12 +24,17 @@ class CandidateConfig(models.Model):
     resume = models.BinaryField(max_length=None, null=True)#If there are errors uploading documents to this column, might need to set editable=True as a parameter
     current_employer = models.ForeignKey(Employer, default=1, on_delete=models.CASCADE, null=True)
 
+    def __str__(self):
+        return self.candidate.__str__()
+
 
 class EmployerConfig(models.Model):
     employer = models.ForeignKey(Employer, default=1, on_delete=models.CASCADE)
     about = models.TextField()
     headquarters = models.CharField(max_length=100)
 
+    def __str__(self):
+        return self.employer.__str__()
```

Comment:
Couldn't this just be return self.employer since we defined the __str__ method in the employer model?

Reply:
Same as my previous comment before. The str method still needs to be called because it won't automatically call the object's str method